src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {
```

Comment:
according to the naming convention enum should be singular
```suggestion
public enum Figure {
```